.vscode/extensions.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Should we push .vscode configs to GH at all?

Reply:
I am not sure, but I think it's necessary stuff, if they pushed it.